### DISCUSSION TITLE
Bug in GitHub profile page.

### DISCUSSION THREAD

**Discussion by BeverlyCode** [community/community#44909] [Other Feature Feedback, Questions, & Ideas] [answered]
Select Topic Area
Bug

If you customise your GitHub profile page pins and select a gist, the link to the gist will not contain the "gist." prefix and thus the link is broken. This used to work and it seems some update has recently broken it.

**Comment by Anmol-Baranwal** [comment, marked as answer]
You can see [here](https://github.com/Anmol-Baranwal). The pin at GitHub profile of a gist is working fine.

- The gist should be public.

Feel free to pin me if there are any further doubts.

**Reply by BeverlyCode** [reply]
It seems to have been fixed now, but I can assure you at the time of reporting this that it was broken through multiple testes.

**Reply by Anmol-Baranwal** [reply]
So, if there are no further doubts. Could you mark my answer as answered and let others know that question has been answered.

**Reply by BeverlyCode** [reply]
yes sorry lol I didn't know I had to do that, I'm not that savvy with GitHub ;)
